samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the shell interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
```

Comment:
Can you also mention how this sample relates to the Zigbee shell library?

Additionally, I see this sample is based on the template one. Does it make sense to make this connection explicit in the doc?

Reply:
Yes. I think it is better to point to the Zigbee shell instead of the Zephyr shell in the intro.
I do not see a value in making a reference between the template and the shell sample: in theory any Zigbee sample is based on the template sample :slightly_smiling_face: